ignite/src/main/java/org/hibernate/ogm/datastore/ignite/IgniteProperties.java

```diff
@@ -29,6 +32,41 @@
 	 */
 	public static final String IGNITE_INSTANCE_NAME = "hibernate.ogm.ignite.instance_name";
 
+	/**
+	 * Configuration property templates for manage read-through and write-through for each entity.
+	 * Example of full string "hibernate.ogm.ignite.persistent_store.Person.read_through=true"
+	 * @see <a href="https://apacheignite.readme.io/docs/persistent-store#read-through-and-write-through">Persistent Store</a>
+	 */
+
+	public static final String IGNITE_CACHE_STORE_CLASS_TEMPLATE = "hibernate.ogm.ignite.persistent_store.%s.cache_store_class";
+	public static final String IGNITE_CACHE_STORE_FACTORY_TEMPLATE = "hibernate.ogm.ignite.persistent_store.%s.cache_store_factory";
+
+	public static final String IGNITE_ALLOWS_TRANSACTION_EMULATION = "hibernate.ogm.ignite.allows_transaction_emulation";
```

Comment:
@schernolyas This should not be an option, it should always be true or false depending if Ignite support transactions or not

Reply:
@DavideD I think same. But ... @Z-z-z-z developed by  another way. You can see classes org.hibernate.ogm.datastore.ignite.transaction.impl.ApplicationServer  and org.hibernate.ogm.datastore.ignite.transaction.impl.IgniteTransactionManagerFactory .  Also, we have a problem with Narayana. These reasons  pushed me to use transction emulation (intergation with native transactions of Ignite). Details of it you can see  https://apacheignite.readme.io/docs/transactions